### PR TITLE
enh(Context): Integrate jotai about User context data

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This migrates the user context integration to Jotai

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

Everything about user data (refresh interval, alias, name, locale, timezone, acknowledgement, downtime and acl) should work as before.

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/front_src/src/__mocks__/@centreon/ui-context.jsx [hunk 1]
```diff
@@ -1,97 +1,6 @@
...
-  useCloudServices,
-  Context,
-  useAcknowledgement,
-};
+export { default as userAtom } from '@centreon/centreon-frontend/packages/ui-context/src/userAtom';
+export { default as aclAtom } from '@centreon/centreon-frontend/packages/ui-context/src/aclAtom';
+export { default as downtimeAtom } from '@centreon/centreon-frontend/packages/ui-context/src/downtimeAtom';
+export { default as refreshIntervalAtom } from '@centreon/centreon-frontend/packages/ui-context/src/refreshIntervalAtom';
+export { default as cloudServicesAtom } from '@centreon/centreon-frontend/packages/ui-context/src/cloudServicesAtom';
+export { default as acknowledgementAtom } from '@centreon/centreon-frontend/packages/ui-context/src/acknowledgementAtom';
```
Comment (Thebarda) [inline www/front_src/src/__mocks__/@centreon/ui-context.jsx:6]: @bdauria I'm not sure about this one but I was not able to remove this "ui-context" mock in order to execute our tests

Comment (bdauria) [reply www/front_src/src/__mocks__/@centreon/ui-context.jsx:6]: you need to use jest's requireActual in order to let what you want from ui-context unmocked

Comment (Thebarda) [reply www/front_src/src/__mocks__/@centreon/ui-context.jsx:6]: The `requireActual` does not work. There is something weird with `@centreon/ui-context` within a Jest environment. Without any mock, when importing all modules from `@centreon/ui-context`, it imports the actual `@centreon/ui` modules

Comment (Thebarda) [reply www/front_src/src/__mocks__/@centreon/ui-context.jsx:6]: Nevermind, I used wrong paths

**Review by jdelpierre — APPROVED** [review]

**Review by Jordan91 — APPROVED** [review]

**Pull request merged**